diff --git a/node_modules/@sushiswap/sushi-data/index.js b/node_modules/@sushiswap/sushi-data/index.js
index 2011f59..a43010b 100644
--- a/node_modules/@sushiswap/sushi-data/index.js
+++ b/node_modules/@sushiswap/sushi-data/index.js
@@ -1,6 +1,15 @@
 'use strict';
 
 const pageResults = require('graph-results-pager');
+const network = require('../../../web/network.json');
+
+// change SubGraph url depending on network id
+ethereum.request({ method: "eth_chainId" }).then(res => {
+	let chainId = Number(res)
+	graphAPIEndpoints.exchange = network[chainId].subGraphExchangeUrl
+}).catch(err => {
+	console.log('err', err)
+})
 
 // TODO: exchange will need to be replaced with new exchange subgraph once it's finished
 const graphAPIEndpoints = {
